components/PostAddForm.js

```diff
@@ -1,7 +1,10 @@
-import { useState, useCallback } from 'react';
+import {useState, useCallback, useRef} from 'react';
 import { useRouter } from 'next/router';
 import Textarea from 'react-textarea-autosize';
 
+import {addComment, addPost} from '../store/actions';
+import {useDispatch, useSelector} from 'react-redux';
```

Comment:
Let's use this order (also, note the formatting around curly braces, and the removal of addComment):

```suggestion
import { useState, useCallback, useRef } from 'react';
import { useDispatch, useSelector } from 'react-redux';
import { useRouter } from 'next/router';
import Textarea from 'react-textarea-autosize';

import { addPost } from '../store/actions';
```